README.md

```diff
@@ -1,2 +1,20 @@
-# era-observability
-Grafana dashboards for zkSync Era
+# Era Observability
+
+This repository contains json sources for Grafana dashboards that are used to observe `ZKSync Era` as well as can be used for any other hyperchain deployment. Dashboards are regularly updated.   
+
+## How to use
```

Comment:
Should you include `--run-observability` flag here?

Reply:
Good point. Rewrote the section given that now we have `--run-observabilit` in place.